task2/src/domen/users/user.router.js

```diff
@@ -25,21 +26,19 @@ import userDatabaseMethods from '../../../repository.js'
 
       })
 
-      userRouter.post('/', (request, response) => {
-            const body = request.body
-
-            if(!Object.keys(request.body).length) {
+      userRouter.post('/', validator, (request, response) => {
```

Comment:
Returning secret credentials like password after creating user is a big no no. Right now when you create a user, it returns password as well. Think about you are hashing it and sending the hashed password. It is a big security issue.